packages/inertia/src/inertia.ts

```diff
@@ -0,0 +1,529 @@
+import {
```

Comment:
Rename this file to `router.ts`, and rename the class to `Router`, but still export it as `Inertia`.